### DISCUSSION TITLE
Welcome to DMPOPIDoR Discussions!

### DISCUSSION THREAD

**Discussion by benjaminfaure** [OPIDoR/DMPOPIDoR#198] [Annonces]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by froggypaule** [comment]
Bonjour @benjaminfaure !
Je m'apprête à installer DMPOPIDoR (pourquoi ? pour avoir une meilleure vision des tables donc de la structuration des données).
Est-oil vraiment nécéssaire de s'en tenir aux versions Ruby 2.4.x, Rails 4.2.x, etc... ou bien les versions les plus récentes sont-elles aussi ok ?
Merci.
